RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -1652,23 +1661,33 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		B387F46F2683FDDB0028701F /* Sources */ = {
+		B3645F3E26E042F2002C490E /* Sources */ = {
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				B39E8118268E832000D31189 /* RCAttributionNetworkAPI.m in Sources */,
-				B3AA35A1268682410007771B /* RCPurchasesAPI.m in Sources */,
-				A5C7008D26AB647B00B2CA64 /* RCOfferingsAPI.m in Sources */,
-				B3083A11269931CF007B5503 /* RCOfferingAPI.m in Sources */,
-				B33CEA9E268BC39D008A3144 /* RCTransactionAPI.m in Sources */,
-				B3AA35A0268682410007771B /* RCEntitlementInfoAPI.m in Sources */,
-				B34D2AA126960A3200D88C3A /* RCIntroEligibilityAPI.m in Sources */,
-				A5A92C7926C716F50038D732 /* RCPackageAPI.m in Sources */,
-				B3AA6234268A7BA900894871 /* RCEntitlementInfosAPI.m in Sources */,
-				B387F47A2683FDEA0028701F /* main.m in Sources */,
-				B3D3C47026856AFF00CB3C21 /* APITester.swift in Sources */,
```

Comment:
Weird this appears as removed 🤔

Reply:
Yeah, this is me removing the target and recreating it. There were a number of build issues and that was how I solved one. You can see right below I added them back in.